source/common/http/http3/conn_pool.cc

```diff
@@ -105,6 +105,12 @@ allocateConnPool(Event::Dispatcher& dispatcher, Random::RandomGenerator& random_
         data.connection_ =
             Quic::createQuicNetworkConnection(h3_pool->quicInfo(), pool->dispatcher(), host_address,
                                               source_address, quic_stat_names, scope);
+        if (data.connection_ == nullptr) {
+          ENVOY_LOG_TO_LOGGER(
+              Envoy::Logger::Registry::getLog(Envoy::Logger::Id::pool), warn,
```

Comment:
Is `warn` going to be too much output in the log? I presume if this happens once, it'll probably happen for every connection attempt. Maybe use `ENVOY_LOG_PERIODIC_TO_LOGGER` or `ENVOY_LOG_EVERY_POW_2_TO_LOGGER`?

Is there any failure metric that increases in this case?

Reply:
@ggreenway updated to **ENVOY_LOG_EVERY_POW_2_TO_LOGGER**. 

As for failure metric, there seems to be no metric to indicate the failure of quic connection creation. From my research, even for HTTP 1/2, there is only a metric  **bind_errors_** which is used to indicate the bind error when failed to create client network connection.